README.md

```diff
@@ -47,8 +60,29 @@ If you don't want to use an IDE like Android Studio, you can build Transportr on
     $ cd Transportr
     $ ./gradlew assembleRelease
 
-License
--------
+## Donations
+
+<table>
+  <tr>
+    <th colspan="3">Thank you for donating</th>
+  </tr>
+  <tr>
+    <td align="center">
+      <a href="https://liberapay.com/Transportr/donate/"><img alt="Donate using Liberapay" src="https://liberapay.com/assets/widgets/donate.svg"></a>
+    </td>
+    <td align="center">Liberapay</td>
+    <td>
+      <a href="https://liberapay.com/Transportr/donate">https://liberapay.com/Transportr/donate</a>
+    </td>
+  </tr>
+  <tr>
+    <td align="center"><a href="bitcoin:12JaQp8zfqRb83JfSwVjH4rZWsZnWRPoyG"><img src="https://transportr.app/img/bitcoin-qr.png" alt="bitcoin:12JaQp8zfqRb83JfSwVjH4rZWsZnWRPoyG" width="128px" height="128px"></a></td>
+    <td align="center">Bitcoin</a></td>
```

Comment:
@grote Do you still want to accept donations via Bitcoin?

Reply:
I don't care either way. The original address should still work and I have still the key, but distributing it to other people is more difficult, so LiberaPay is probably better.